packages/core/src/physics/joint/JointLimits.ts

```diff
@@ -41,7 +41,7 @@ export class JointLimits {
 
   set min(value: number) {
     if (value > this._max) {
-      throw new Error("Min limit must be less than max limit");
+      this._max = value;
```

Comment:
Why does it throw error when setting `max` but not here?
L27

Reply:
Need discuss. The default value of min and max is 0, it means it will throw error when user set any value, i think it is not comfortable